public/locales/en-US/send.ftl

```diff
@@ -117,3 +126,30 @@ passwordIsSet = Password set
 maxPasswordLength = Maximum password length: { $length }
 # A short status message shown when there was an error setting the password
 passwordSetError = This password could not be set
+pageHeaderCredits = from the makers of Firefox
+addFilesButton = Add file(s)
+uploadFilesButton = Send
+uploadFileProgress = Sending
+uploadDropDragMessage = Drop files here
+uploadDropClickMessage = or click to select a file
```

Comment:
Same as above. It would help to understand why this starts lowercase. Also, do we really need to split this sentence in two parts?